cms_form/static/src/js/ajax.js

```diff
@@ -0,0 +1,65 @@
+odoo.define('cms_form.ajax', function (require) {
+    'use strict';
+
+    var core = require('web.core'),
+        animation = require('website.content.snippets.animation');
+
+    animation.registry.CMSFormAjax = animation.Class.extend({
+        selector: ".cms_form_wrapper form[data-ajax]",
+        start: function () {
+            var data = this.$el.data('form');
+            this.$el.on('submit', this.proxy('submit_form'));
+            if (data.ajax_onchange) {
+                this.$el.on('change', this.proxy('submit_form'));
+            };
+        },
+        submit_form: function (ev) {
+            var $this = this.$el,
+                data = $this.data('form');
+            var items_iterator = document.evaluate(
```

Comment:
do we really need this? I'd:
1. blur current results while searching and show a spinner
2. submit and get the full HTML back
3. replace HTML remove blurring and spinner (just turn on / off a class of course)